src/CodingMilitia.PlayBall.Shared.StartupTasks/CodingMilitia.PlayBall.Shared.StartupTasks.csproj

```diff
@@ -4,4 +4,12 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="AspNetCore.AsyncInitialization" Version="1.1.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\CodingMilitia.PlayBall.GroupManagement.Data\CodingMilitia.PlayBall.GroupManagement.Data.csproj" />
```

Comment:
As this will be a shared project, it shouldn't have dependencies on `GroupManagement` specifics.